src/test/java/com/microsoft/sqlserver/jdbc/fedauth/PooledConnectionTest.java

```diff
@@ -435,29 +435,8 @@ public void testDSPooledConnectionAccessTokenCallbackClassExceptions() throws Ex
 
         // User/password is not required for access token callback
         AbstractTest.updateDataSource(accessTokenCallbackConnectionString, ds);
-        ds.setAccessTokenCallbackClass(AccessTokenCallbackClass.class.getName());
-        ds.setUser("user");
         SQLServerPooledConnection pc;
-
-        // Should fail with user set
-        try {
-            pc = (SQLServerPooledConnection) ds.getPooledConnection();
-            fail(TestResource.getResource("R_expectedFailPassed"));
-        } catch (SQLServerException e) {
-            assertTrue(e.getMessage().matches(TestUtils.formatErrorMsg("R_AccessTokenCallbackWithUserPassword")));
-        }
-
-        ds.setUser("");
-        ds.setPassword(UUID.randomUUID().toString());
-
-        // Should fail with password set
-        try {
-            pc = (SQLServerPooledConnection) ds.getPooledConnection();
-            fail(TestResource.getResource("R_expectedFailPassed"));
-        } catch (SQLServerException e) {
-            assertTrue(e.getMessage().matches(TestUtils.formatErrorMsg("R_AccessTokenCallbackWithUserPassword")));
-        }
-

```

Comment:
We can test for the opposite, instead. Let's test that if user/pwd is set, it passes.

Reply:
Updated the test case, please review.